modules/data/about_addons.components.json

```diff
@@ -21,5 +21,23 @@
         "selectorData": "back-button",
         "strategy": "class",
         "groups": []
+    },
+
+    "enabled-theme-title": {
+        "selectorData": ".theme-enabled-section .addon-name",
+        "strategy": "css",
+        "groups": []
+    },
+
+    "find-more-themes-button": {
```

Comment:
nit picking up these two selectors can be made into 1 with the label, but up to u!

Reply:
They *could* but it's not like they're "good buddies" or anything. If they were two buttons that we always saw together, I would have, believe me. I love me some labels.